src/app/containers/Figure/index.jsx

```diff
@@ -11,10 +11,18 @@ const renderCopyright = copyright =>
 
 const renderCaption = block => (block ? <Caption block={block} /> : null);
 
-const FigureContainer = ({ src, alt, ratio, copyright, captionBlock }) => (
+const FigureContainer = ({
+  alt,
+  copyright,
+  captionBlock,
+  height,
+  ratio,
+  src,
+  width,
+}) => (
   <Figure>
     <ImagePlaceholder ratio={ratio}>
-      <Image alt={alt} src={src} />
+      <Image alt={alt} src={src} width={width} height={height} />
```

Comment:
```suggestion
      <Image alt={alt} src={src} width={width} />
```
This is because adding the height too makes the image stretch:
With `height` attribute:
<img width="511" alt="screen shot 2018-12-19 at 16 38 42" src="https://user-images.githubusercontent.com/3028997/50234026-a5a7b300-03ac-11e9-8281-23f29184b238.png">
Without `height` attribute:
<img width="506" alt="screen shot 2018-12-19 at 16 41 03" src="https://user-images.githubusercontent.com/3028997/50234163-f1f2f300-03ac-11e9-802a-8400d8555407.png">